doc_requirements.txt

```
@@ -0,0 +1,3 @@
+Sphinx
+git+https://github.com/pydata/pydata-sphinx-theme.git@master
+breathe
```

I prefer to pin versions

```suggestion
Sphinx==4.4.0
pydata-sphinx-theme==.0.8.0
breathe==4.31.0
```